test/regress/regress1/quantifiers/prenex-scholl-smt08_RNDPRE_RNDPRE_4_6.smt2

```diff
@@ -0,0 +1,20 @@
+; EXPECT unsat
```

Comment:
```suggestion
; EXPECT: unsat
```

Reply:
Note this is actually unecessary, since there is also (set-info :status unsat) in the benchmark below.